templates/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,147 @@
+WITH crash_ping_agg AS (
```

Comment:
Nit: I'm not sure these result sets should be suffixed with `_agg` since there are no aggregations there?